src/libraries/Native/Windows/CMakeLists.txt

```diff
@@ -11,15 +11,19 @@ SET (CMAKE_ASM_MASM_FLAGS                   "${CMAKE_ASM_MASM_FLAGS} /ZH:SHA_256
 SET (CMAKE_INSTALL_PREFIX                   $ENV{__CMakeBinDir})
 SET (CMAKE_INCLUDE_CURRENT_DIR              ON)
 SET (CMAKE_SHARED_LIBRARY_PREFIX            "")
-SET (__LinkArgs $ENV{__LinkArgs})
+
+set(__SharedLinkArgs)
```

Comment:
I assume `__SharedLinkArgs` are to separate flags applicable only when building a dll, right?

Reply:
Right, the build was generating a lot of warnings when I've applied all the options to the static library, as those are linker options and not librarian options.